docs/rules/assertion-arguments.md

```diff
@@ -0,0 +1,61 @@
+# Enforce passing correct arguments to assertions
+
+Enforces passing the right number of arguments to assertion methods like `t.is()`. This rule can optionally also enforce or forbid the use of assertion messages.
+
+Assertion messages are optional arguments that can be given to any assertion call to improve the error message, should the assertion fail.
+
+
+## Fail
+
+```js
+import test from 'ava';
+
+test(t => {
+	t.is(value); // Not enough arguments
+	t.is(value, expected, message, extra); // Too many arguments
+});
+
+/* eslint ava/assertion-arguments: ["error", "always"] */
+test(t => {
+	t.true(array.indexOf(value) !== -1);
+});
+
+/* eslint ava/assertion-arguments: ["error", "never"] */
+test(t => {
+	t.true(array.indexOf(value) !== -1, 'value is not in array');
+});
+```
+
+
+## Pass
+
+```js
+import test from 'ava';
+
+test(t => {
+	t.is(value, expected);
+	t.is(value, expected, message);
+});
+
+/* eslint ava/assertion-arguments: ["error", "always"] */
+test(t => {
+	t.true(array.indexOf(value) !== -1, 'value is not in array');
+});
+
+/* eslint ava/assertion-arguments: ["error", "never"] */
+test(t => {
+	t.true(array.indexOf(value) !== -1);
+});
+```
+
```

Comment:
You should talk to @wooorm. He has already done that. https://github.com/wooorm/remark-lint